models/media_ads/snowplow_media_player_media_ads.sql

```diff
@@ -52,6 +52,7 @@ new_media_ad_views as (
     a.media_ad_id,
 
     a.platform,
+    a.app_id,
```

Comment:
This comes with a breaking change where previous users would have to do a full refresh, as ultimately this changes the whole structure of the table. As this is not a requirement for the app so we should probably not add it here. Let's check with product.